2021-10-13-github-templates.md

```diff
@@ -0,0 +1,123 @@
+an issue.
+
+## Proposal:
+
+Come up with a consistent set of Github templates. The templates should include prompts
+only the ESSENTIAL information we require. We want to make sure that we don't ask for more
+information than we require - as this will discourages contributions.
+
+## Templates:
+
+### Pull Requests
+
+```
+    <!--- Before we start... -->
+    <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+    <!--- Did you update any relevant documentation -->
+    <!--- Did you add tests to cover new changes -->
+
+    ## Description
+    <!--- Describe your changes in detail -->
+    <!--- Please read the README before submitting pull requests for this project. -->
+
+    ## Motivation and Context
+    <!--- Why is this change required? What problem does it solve? -->
+    <!--- If it fixes an open issue, please link to the issue here. -->
+
+    ## How Has This Been Tested?
+    <!--- Include any relevant details about your testing environment and the steps you followed -->
+
+    ## Screenshots (if appropriate):
+
+    **I will abide by the [code of conduct](<LINK TO CODE OF CONDUCT HERE>)**
+```
+
+
+### Issues: Bug Reports
+
+```
+    <!--- Before we start... -->
+    <!--- Did you check the documentation for answers? -->
+    <!--- Did you make sure that this bug has not already been reported? -->
+
+    ## Expected Behavior
+    <!--- Tell us what should happen -->
+
+    ## Actual Behavior
+    <!--- Tell us what happens instead -->
+
+    ## Possible Fix
+    <!--- Optionally suggest a fix or reason for the bug -->
+
+    ## Steps to Reproduce
+    <!--- Provide a link to a live example, or an unambiguous set of steps to -->
+    <!--- reproduce this bug. Include code to reproduce, if relevant -->
+    1.
+    2.
+    3.
+    4.
+
+    ## Additional Information
+    <!--- Include any relevant details about your environment (branch, browser, OS) -->
+    <!--- and the bug (screenshots, logs, etc) -->
+
+    **I will abide by the [code of conduct](<LINK TO CODE OF CONDUCT HERE>)**
+```
+
+### Issues: Feature Requests
+
+```
+    <!--- Before we start... -->
+    <!--- Did you check the documentation for this feature? -->
+    <!--- Did you make sure that this feature has not already been requested? -->
+
+    ## Description
+    <!--- Provide a description of the change or addition you are proposing -->
+
+    ## Possible Implementation
+    <!--- Optionally suggest an idea for implementing addition or change -->
+
+    ## Resources:
+    <!--- If you have resources related to the implementation or research for this feature, add them here. -->
+    <!--- Include any mockup idea related to the requested feature if it applies. -->
+
+    **I will abide by the [code of conduct](<LINK TO CODE OF CONDUCT HERE>)**
+```
+
+## Final thoughts:
+
+For the sake of simplicity we would prefer to have a single set of templates. We might
```

Comment:
For the sake of simplicity, it might be a good idea to have a one-liner to quickly implement the templates in a project: 

```
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/fastruby/install/templates/install.sh)"
```